src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java

```diff
@@ -48,6 +49,7 @@ public FeedbackQuestionAttributes(FeedbackQuestion fq) {
         this.questionMetaData = fq.getQuestionMetaData();
         this.questionNumber = fq.getQuestionNumber();
         this.questionType = fq.getQuestionType();
+        this.questionIsCompulsory = fq.isQuestionIsCompulsory();
```

Comment:
This causes existing questions to throw an error. You need to handle cases where the new field is not present in the datastore.